src/store/ducks/opensrp/hierarchies/index.tsx

```diff
@@ -55,6 +55,9 @@ export const DESELECT_ALL_NODES = 'opensrp/locations/hierarchy/DESELECT_ALL_NODE
 // /** action to auto-append/ auto-modify the selected attribute of nodes in a tree */
 export const AUTO_SELECT_NODES = 'opensrp/locations/hierarchy/AUTO_SELECT_NODES';
 
+/** action to fetch updated children to ensure map and jurisdiction table are in sync */
+export const FETCH_MAP_CURRENT_PARENT = 'opensrp/locations/hierarchy/FETCH_MAP_CURRENT_PARENT';
```

Comment:
is this being used somewhere?

Reply:
@p-netm  addressed via https://github.com/onaio/reveal-frontend/pull/1083/commits/6ca235ed7f320553ff768af85161ae54f5f39023